.readthedocs.yaml

```
@@ -16,7 +17,14 @@ build:
       - playwright install chromium
```

```suggestion
      - playwright install --with-deps chromium
```

This should get rid of the missing dependencies error you have been hitting